benchmark/src/main/java/gov/cms/qpp/conversion/SpeedTest.java

```diff
@@ -0,0 +1,73 @@
+package gov.cms.qpp.conversion;
+
+import com.google.common.collect.Lists;
+
+import java.nio.file.Paths;
+import java.util.List;
+
+/**
+ * A suite of conversions where the time is averaged and then averaged again to give a relatively stable time in milliseconds
+ * that a conversion takes.
+ */
+public class SpeedTest {
+	private static final int SETS_PER_SUITE = 10;
+	private static final int TESTS_PER_SET = 20;
```

Comment:
this number is probably not high enough. When running in -server mode, the jvm invocation threshold for method compiling is 10k. In -client it's 1k.